### PR TITLE
chore: Remove SecurityLabels funtionality - now a dependency

### PR DESCRIPTION
**Pull request by gnikolov95**

### CODE REVIEW

File: telicent_lib/adapter.py [hunk 1]
```diff
@@ -140,6 +139,8 @@ def __init__(self, target: DataSink, adapter_function: RecordAdapter,
         :param name: The name of the Adapter
         :type name: str
         :param source_name: The name of the data source, used in the startup banner
+        :param pbac_obj: An instance of Security policy model populated with data
+        :type object
```
Comment (danieldaviestelicent) [inline telicent_lib/adapter.py:143]:
```suggestion
        :type pbac_job: object
```

Just to match the other styling

**Review by danieldaviestelicent — CHANGES_REQUESTED** [review]
Just one small niggle with a doc string but overall lgtm as an approach. 

I think we need to ask @tatelicent to approve too, he was going to look over the other repo as well.

**Review by danieldaviestelicent — DISMISSED** [review]
I think we just need @tatelicent to be happy with this too

File: .pre-commit-config.yaml
Comment (danieldaviestelicent) [inline .pre-commit-config.yaml:1]: Can we make this change with a seperate PR, or can you not commit at all without it?

File: docs/security-label-modules.md
Comment (danieldaviestelicent) [inline docs/security-label-modules.md:1]: What is the integration actually doing, is it possible to document how it is used in telicent-lib? Some kind of, "here's how you label your incoming data" type thing?

File: pyproject.toml [hunk 2]
```diff
@@ -20,7 +20,7 @@ dependencies = [
     "confluent-kafka==2.3.0",
     "opentelemetry-api==1.20.0",
     "opentelemetry-sdk==1.20.0",
-    "pydantic==2.5.2",
+    "telicent-label-builder==0.1.0"
```
Comment (danieldaviestelicent) [inline pyproject.toml:23]: Where doe this dependency actually get used? Is it only in tests?

Comment (gnikolov95) [reply pyproject.toml:23]: Yes it is only test, perhaps it should be moved to extra dev.

File: .pre-commit-config.yaml
Comment (gnikolov95) [reply .pre-commit-config.yaml:1]: pre-commit will keep failing locally unless this is added. But i can disable it and just make this change in a separate PR is it not a problem.

File: docs/security-label-modules.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+A validated instance of a model can subsequently be converted into a security label, which can then be appended as a 
+header to a record. It is important to note that telicent-lib does not enforce any of these operations; 
+the implementation is entirely at the discretion of the user. The label-builder module provides users with the 
+functionality to define and implement their own models. For guidance purposes, the label-builder currently includes 
+an implementation of the TelicentModel. Label-builder is not a dependency of telicent-lib.
+
+```python
+data_header = {
+            "apiVersion": "v1alpha",
+            "specification": "UKIC v3.0",
```
Comment (tatelicent) [inline docs/security-label-modules.md:18]:
```suggestion
            "specification": "v3.0",
```

File: tests/test_adapter.py [hunk 4]
```diff
@@ -40,7 +48,7 @@ def setUp(self) -> None:
             ('Exec-Path', b'Automatic Adapter-to-In-Memory List'), ('Request-Id', b'List:uuid4'), ('traceparent', b'')
         ]
 
-        self.test_policy = {
+        self.test_data_header = {
             "apiVersion": "v1alpha",
             "specification": "UKIC v3.0",
```
Comment (tatelicent) [inline tests/test_adapter.py:53]:
```suggestion
            "specification": "v3.0",
```

**Review by danieldaviestelicent — DISMISSED** [review]

**Review by danieldaviestelicent — APPROVED** [review]

**Pull request merged**